### DISCUSSION TITLE
News Cache [2022-09-30]

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#918] [Announcements]
📰 Great news this week, as usual 🙌 

- Some excellent [improvements and fixes](https://vercel.com/changelog/september-2022-papercuts) were released, including:
  - 🧰 Vercel CLI [v28.4.5](https://github.com/vercel/vercel/releases/tag/vercel%4028.4.5) with bug fixes and improved JSON parsing
  - 💻 New system environment variable, [VERCEL_GIT_PREVIOUS_SHA](https://vercel.com/docs/concepts/projects/environment-variables#system-environment-variables), is now available in the [Ignored Build Step](https://vercel.com/docs/concepts/projects/overview#ignored-build-step) to compare changes against the SHA of the last successful deployment.
  - 📝 [CLI docs](https://vercel.com/docs/cli) revamp 
  - 📓 Improved [deployment logs](https://vercel.com/docs/concepts/deployments/logs)
  - and many more that you can read about in the [changelog](https://vercel.com/changelog/september-2022-papercuts)!
- ▲ You can now [access Vercel brand assets and guidelines](https://vercel.com/changelog/easily-access-vercel-brand-assets-and-guidelines) by right-clicking the logo anywhere on the platform.
- 🧐 [Larger monorepos are now supported](https://vercel.com/changelog/improved-monorepo-support-with-increased-projects-per-repository) for Pro users, allowing for up to 60 projects per git repository.

---

If you have any news, wins, or helpful tips from this week, please share them below!

**Comment by lesenelir** [comment]
🐰🐰
